storybook/src/elements/nuxeo-card/nuxeo-card.stories.js

```diff
@@ -10,11 +10,11 @@ const ICONS = {
   none: '',
 };
 
-storiesOf('UI/nuxeo-card', module).addElement('nuxeo-card', ({ knobs }) => {
-  const { heading, collapsible, icon, opened } = knobs({
-    heading: { value: 'Heading' },
-    icon: { type: 'select', options: ICONS },
-  });
+storiesOf('UI/nuxeo-card', module).add('nuxeo-card', () => {
+  const heading = text('Heading', 'Heading');
+  const icon = select('Icon', ICONS, ICONS.attachment);
+  const collapsible = boolean('Collapsible', false);
+  const opened = boolean('Opened', false);
   return html`
     <style>
       nuxeo-card {
```

Comment:
Not sure if we need this kind of styling since we have this: https://github.com/nuxeo/nuxeo-elements/blob/master/storybook/.storybook/nuxeo-demo-theme.js

Reply:
ok, removing style.